### PR TITLE
Enhance PTL MoM

### PR DESCRIPTION
**Pull request by sakshamgarg**
#### Describe Bug or Feature

Enhance the current PTL MoM framework

#### Describe Your Change

Added 4 more APIs in MoM class and made changes in the test cases
1. get_formed_path
2. rm
3. isfile
4. listdir

### CODE REVIEW

**Comment by bhroam** [comment]
Why do we need these in mom?  We have them in DshUtils.

File: test/fw/ptl/lib/ptl_mom.py [hunk 1]
```diff
@@ -169,6 +169,78 @@ def __init__(self, server, name=None, attrs={}, pbsconf_file=None,
...
+        :param argv: argument variables
+        :type argv: str
+        :returns: A string of formed path
+        """
+
+        if len(argv) == 0:
+            return None
+        return os.path.join(*argv)
+
+    def rm(self, hostname=None, path=None, sudo=False, runas=None,
```
Comment (vishwaks) [inline test/fw/ptl/lib/ptl_mom.py:183]: we can remove hostname argument for this API also.

**Review by vishwaks — CHANGES_REQUESTED** [review]

**Review by vishwaks — APPROVED** [review]
Looks good to me.

**Review by npadole20 — APPROVED** [review]

**Comment by bayucan** [comment]
The changes looked fine. Is there an accompanying test logs for this on the Linux side?

**Review by bayucan — APPROVED** [review]
Looks good. I've seen the test results.

**Pull request merged**